docs/metadata.rst

```diff
@@ -21,9 +21,12 @@ the :ref:`Standard JSON Interface<compiler-api>`.
 
 You have to publish the metadata file to IPFS, Swarm, or another service so
 that others can access it. You create the file by using the ``solc --metadata``
```

Comment:
```suggestion
that others can access it. You create the file by using the ``solc --metadata`` command together with the ``--output-dir`` parameter. Without it, it will only be written to standard out.
```

I think it's important to mention both in the same sentence, otherwise ppl might try it and only read the ``--ouput-dir` part later.

Reply:
Updated it in the same sentence.